erpnext/hr/doctype/job_offer/test_job_offer.py

```diff
@@ -13,8 +13,11 @@
 
 class TestJobOffer(unittest.TestCase):
 	def test_job_offer_creation_against_vacancies(self):
-		create_staffing_plan(staffing_details=[{
-			"designation": "Designer",
+		create_staffing_plan(name="Test 2", 
+		from_date = nowdate(),
+		to_date = add_days(nowdate(), 10),
+		staffing_details=[{
+			"designation": "Researcher",
```

Comment:
```suggestion
		create_staffing_plan(
			name="Test 2", 
			from_date = nowdate(),
			to_date = add_days(nowdate(), 10),
			staffing_details=[{
				"designation": "Researcher",
```
indent below lines as well